<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_titre">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Programme pour vérifier la loi de Fitts</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>521</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Rappel :</string>
    </property>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_b">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>110</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.200000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_a">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.200000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_choix">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Choix de a et b :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_a">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Variable a</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_b">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>110</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Variable b</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_formule">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Formule utilisée :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_formuleFitts">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>T = a + b.log2(D/W+1)</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>271</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuration du test :</string>
    </property>
    <widget class="QSpinBox" name="spinBox_nb">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_taille_min">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_taille_max">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>50</number>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="value">
      <number>150</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_nb">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre de cibles :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_taille_min">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Taille minimum cible :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_taille_max">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Taille maximum cible :</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_quitter">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>400</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_demarrer">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Démarrer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
